### PR TITLE
v122

### PR DESCRIPTION
**Pull request by Thorin-Oakenpants**

### CODE REVIEW

**Comment by mid-kid** [comment]
Aren't there any new settings related to [Global Privacy Control](https://globalprivacycontrol.org/)?

**Comment by Thorin-Oakenpants** [comment]
> Aren't there any new settings related to [Global Privacy Control](https://globalprivacycontrol.org/)?

the PR isn't finished - but as for GPC, the pref has been there for a while, what they have done is expose it in the UI

File: user.js [hunk 1]
```diff
@@ -686,8 +687,32 @@ user_pref("privacy.cpd.cookies", false);
...
+/* 4001: enable FPP in PB mode [FF114+]
+ * [NOTE] In FF119+, FPP for all modes (7106) is enabled with ETP Strict (2701) ***/
+   // user_pref("privacy.fingerprintingProtection.pbmode", true); // [DEFAULT: true FF118+]
+/* 4002: set global FPP overrides [FF114+]
+ * Controls what protections FPP uses globally, including "RFPTargets" (despite the name these are
+ * not used by RFP) e.g. "+AllTargets,-CSSPrefersColorScheme" or "-AllTargets,+CanvasRandomization"
+ * [NOTE] Be aware that not all RFP protections are necessarily in RFPTargets
+ * [WARNING] Not recommended. Either use RFP or FPP at defaults
+ * [1] https://searchfox.org/mozilla-central/source/toolkit/components/resistfingerprinting/RFPTargets.inc ***/
+   // user pref("privacy.fingerprintingProtection.overrides", "");
```
[the remark this answers is not in the document]
Comment (Thorin-Oakenpants) [reply user.js:711]: actually, I'm trying to find a better bugzilla for
``
   1826408 - restrict fonts to system (kBaseFonts + kLangPackFonts) (Windows, Mac, some Linux)
```

specifically I want the one that hooks it up to FPP

File: user.js [hunk 2]
```diff
@@ -686,8 +687,32 @@ user_pref("privacy.cpd.cookies", false);
...
+***/
+user_pref("_user.js.parrot", "1400 syntax error: the parrot's bereft of life!");
+/* 4001: enable FPP in PB mode [FF114+]
+ * [NOTE] In FF119+, FPP for all modes (7106) is enabled with ETP Strict (2701) ***/
+   // user_pref("privacy.fingerprintingProtection.pbmode", true); // [DEFAULT: true FF118+]
+/* 4002: set global FPP overrides [FF114+]
+ * Controls what protections FPP uses globally, including "RFPTargets" (despite the name these are
+ * not used by RFP) e.g. "+AllTargets,-CSSPrefersColorScheme" or "-AllTargets,+CanvasRandomization"
+ * [NOTE] Be aware that not all RFP protections are necessarily in RFPTargets
+ * [WARNING] Not recommended. Either use RFP or FPP at defaults
```
Comment (sertonix) [inline user.js:709]: I have the feeling that quite a few RFPTargets can be used without any disadvantage. For example most spoofed boolean like `+CSSPrefersColorScheme`.

What are you thought about that?

Comment (Thorin-Oakenpants) [reply user.js:709]: > Am I missing something

line 709

**Pull request merged**